static/app/utils/profiling/hooks/useFunctions.spec.tsx

```diff
@@ -22,11 +23,15 @@ const selection: PageFilters = {
 
 function TestContext({children}: {children: ReactElement}) {
   const {organization} = useMemo(() => initializeOrg(), []);
+  // ensure client is rebuilt on each render otherwise caching will interfere with subsequent tests
```

Comment:
no this is intentional to rebuild it for each test... for context i had `client` scoped to the module, not the component and had some tests fail since the `queryKey` would hit cache first.

another example of this footgun... in `useProfileEvents` the `client` is scoped to the module... 
![image](https://user-images.githubusercontent.com/7349258/217607466-69c8a160-c109-42c8-b4bd-05fb23b6b513.png)

i'm gonna fix that in a separate PR and see if our test suite is actually accurate or not